### PR TITLE
LPS-116044 Add Screen Navigation category for Account Groups

### PR DESCRIPTION
**Pull request by alee8888**
https://issues.liferay.com/browse/LPS-116044

@drewbrokke @jonmak08

### CODE REVIEW

**Comment by alee8888** [comment]
ci:test:sf

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/frontend/taglib/servlet/taglib/AccountGroupAccountsScreenNavigationCategory.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	public String getScreenNavigationKey() {
+		return AccountScreenNavigationEntryConstants.
+			SCREEN_NAVIGATION_KEY_ACCOUNT_GROUP;
+	}
+
+	@Override
+	public boolean isVisible(
+		User user, AccountGroupDisplay accountGroupDisplay) {
+
+		if (accountGroupDisplay == null) {
```
Comment (pei-jung) [inline modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/frontend/taglib/servlet/taglib/AccountGroupAccountsScreenNavigationCategory.java:74]: Does this work? Looks like `accountGroupDisplay` will never be null. Maybe check for `accountGroupId` instead?

File: modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/frontend/taglib/servlet/taglib/AccountGroupInformationScreenNavigationCategory.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+	public String getScreenNavigationKey() {
+		return AccountScreenNavigationEntryConstants.
+			SCREEN_NAVIGATION_KEY_ACCOUNT_GROUP;
+	}
+
+	@Override
+	public boolean isVisible(
+		User user, AccountGroupDisplay accountGroupDisplay) {
+
+		if (accountGroupDisplay == null) {
```
Comment (pei-jung) [inline modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/frontend/taglib/servlet/taglib/AccountGroupInformationScreenNavigationCategory.java:74]: If this is used for both add and edit, then this tab is always visible. If so, we don't need to define the `isVisible` method.

**Comment by pei-jung** [comment]
See https://github.com/pei-jung/liferay-portal/pull/656.

**Pull request closed without merging**